staticjinja/reloader.py

```diff
@@ -39,16 +83,45 @@ def event_handler(self, event_type, src_path):
 
         """
         filename = os.path.relpath(src_path, self.searchpath)
+        # Avoid vim super temporary file
+        if filename.endswith('4913'):
+            return
```

Comment:
We should remove this. You can disable temporary files by the way, which is what I do (rarely does my computer crash to warrant the hassle of dealing with them).